examples/span_logging.py

```diff
@@ -0,0 +1,96 @@
+""" Example usage of the BasicTracer implementation which log spans to the console.
+
+Ensure the following pip packages are installed:
+
+    - opentracing
+    - basictracer
+
+Run with the command:
+
+    python3 examples/span_logging.py
+
+Example output:
+
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 0}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 1}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 2}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.01e+00 S, {'i': 3}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 4}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 5}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 6}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 7}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 8}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.loop[1.00e+00 S, {'i': 9}]: {'message': 'Sleeping for 1 second'}
+    [DEBUG   ] span_logging.main[1.00e+01 S]: finished
```

Comment:
I am not sure why this format of logging the span is particularly useful. It doesn't capture span IDs so you cannot see causality.

Reply:
This is true, I'll take a crack at making this more useful and pretty.